cmd/example/client.go

```diff
@@ -0,0 +1,115 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This example is adapted from:
+// https://github.com/aspenmesh/istio-client-go/blob/4de6e89009c427dbc602b0c6bbdc8840ef1905e6/cmd/example-client/client.go
```

Comment:
@geeknoid is this the correct way to attribute partially coped code in Istio. We normally change "Copyright" to "Portions Copyright", not sure what practices Istio follows.

Reply:
We don't have a policy for this kind of situation. 

@sdake, would you have some advice here?

Reply:
Not a big deal in this case as I'm author/creator of the OS repo that I'm copying this from but it would be good to know in general the overall policy.